### ISSUE TITLE
[Website]: Luminosity ratio of 'Storybook.js' and 'Hexo' link text is 4.0:1with respect to background which is less than 4.5:1 on Adaptive Cards page.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6420]
### Target Platforms

NodeJS

### SDK Version

 Adaptive Cards (adaptivecards-ci.azurewebsites.net) 6/22/2021

### Application Name

Website

### Problem Description

[ADO 33896871](https://microsoft.visualstudio.com/OS/_workitems/edit/33896871)

**Repro Steps:**
1. Launch the URL: Adaptive Cards (adaptivecards-ci.azurewebsites.net)
2. Navigate to the footer section of Adaptive Cards web page.
3. Observe the issue whether luminosity ratio of 'Storybook.js' and 'Hexo' link text is less than 4.5:1 or not.

**Actual Result:**
Luminosity ratio of 'Storybook.js' and 'Hexo' link text is 4.0:1 which is less than 4.5:1 on Adaptive Cards page.

**Expected Result:**
Luminosity ratio of 'Storybook.js' and 'Hexo' link text should be greater than or equal to 4.5:1.

### Card JSON

```JSON
No card
```
